client/src/components/Picture/UploadPic.js

```diff
@@ -249,31 +249,35 @@ const UploadPic = ({ cameraIconSize, gtmPrefix, user }) => {
             {uploadError ? (
               <h3>{uploadError}</h3>
             ) : (
-              <ReactCrop
-                src={photoURL}
-                crop={crop}
-                minHeight={250}
-                minWidth={250}
-                circularCrop={true}
-                ruleOfThirds
-                keepSelection={true}
-                onImageLoaded={imageLoaded}
-                onChange={(newCrop) => setCrop(newCrop)}
-              />
-            )}
+                <ReactCrop
```

Comment:
Not sure if my current linter has the right style? What is the guidelines for indentation here?

Reply:
it seems my side 235-238, 252-263 and 266-280 has one tab less than yours @linadinh2602  (while I don't know if there is a specification in our repos)